frontend/src/components/search/SearchFilterAccordion/SearchFilterAccordion.tsx

```diff
@@ -43,20 +44,42 @@ export interface FilterOptionWithChildren {
   children: FilterOption[];
 }
 
+const isSectionAllSelected = (
```

Comment:
these should probably be moved to a utils file, but getting them out of the component is a good first step. Saving bigger refactors for https://github.com/HHS/simpler-grants-gov/issues/3331